aiida/orm/node/process/calculation/calcjob.py

```diff
@@ -1593,12 +1586,12 @@ def _get_calculation_info_row(cls, res, projections, times_since=None):
         d = copy.deepcopy(res)
 
         try:
-            prefix = 'calculation.job.'
+            prefix = 'node.process.calculation.calcjob.'
             calculation_type = d['calculation']['type']
             calculation_class = get_plugin_type_from_type_string(calculation_type)
             module, class_name = calculation_class.rsplit('.', 1)
 
-            # For the base class 'calculation.job.JobCalculation' the module at this point equals 'calculation.job'
+            # For the base class 'calculation.job.CalcJobNode' the module at this point equals 'calculation.job'
```

Comment:
This comment is outdated? (`calculation.job.CalcJobNode` -> `node.process.calculation.calcjob.CalcJobNode`?)